index.js

```diff
@@ -31,14 +47,33 @@ class Chapter {
   }
 
   skip(array, n) {
-    const output = array.slice();
-    output.splice(0, n);
-    return output;
+    if (typeof array === 'number') {
+      const output = this.result.slice();
+
+      output.splice(0, array);
+      this.result = output;
+      return this;
+
+    } else {
```

Comment:
since we are returning in the previous if - the `else` block is unnecessary here. In case if we go into `if`, this code won't be executed, since the function would already return.

Reply:
removed `else` block